modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -252,11 +268,21 @@ export default {
     // Cleanly namespace it so we don't conflict with other uses and instances
     const CustomPlaceholder = Placeholder.extend();
     const extensions = [
-      StarterKit.configure({
-        document: false,
-        heading: false,
-        listItem: false
-      }),
+      BlockQuote,
```

Comment:
@haroun I haven't see any starter-kit specifics (configuration wise), please validate that.